examples/supported_use_cases/records_in_cqrs.cs

```diff
@@ -0,0 +1,19 @@
+using LeanCode.Contracts;
+
+public record DTO1(int Property);
+public record DTO2(string Property);
+
+public class Command : ICommand
```

Comment:
I meant that we should check if this will work:
```
public record Command(DTO1 DTO1) : ICommand;
```
et al. :)

Reply:
Huh, didn't think about this one, but it looks like it works as well